notebooks/testgrid_feature_confirmation.ipynb

```diff
@@ -0,0 +1,1871 @@
+{
```

Comment:
<p>Hi Michael, </p><p> </p><p>Do we have a description of above features?</p><p>Can we add a link if it is available somewhere else?&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
@SankBad I'm not sure if there is a complete description on these.  I think we will have to dig into them and see what is there to generate a description. The closest thing I could find was the [config docs](https://github.com/GoogleCloudPlatform/testgrid/blob/master/config.md) on the test grid repo.

Reply:
got it :+1: , Thanks